wiki/Rules/Code_of_conduct_for_modding_and_mapping/en.md

```diff
@@ -4,58 +4,33 @@ The **code of conduct for modding and mapping** is a set of rules and guidelines
-
-- Treat all suggestions as valid and equally important until you have thoroughly inspected and tested them — even minor stylistic changes or alternative viewpoints can help improve the quality of a beatmap
-- As with making a mod post, be sure to explain yourself as much as possible. State what you changed, and if you didn't accept the suggestion for some reason, explain why and what you did instead.
-- When declining a suggestion, remain calm and professional, say what needs to be said in order to explain yourself and **leave it at that**. No matter how impolite, confusing or unhelpful the suggestion was, getting angry and responding with hostility, sarcasm, etc. will not help the situation.
-- [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (BN) and [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (NAT) members will review previous mods before placing any icons and may want to initiate or revisit discussions during their checks. Any assistance you can provide as to why you did or didn't do something takes very little effort, and could be a huge help to all parties later on.
+It's necessary to give a proper response to a mod post. Not only will it make it easier for future mods to be tailored to your concept or style, but it will also help the modder improve their modding ability.
 
-## Further exchange and discussion
+In order for this exchange to be productive, it is important to consider the following:
 
-Sometimes, a single mod and response isn't enough to bring all parties to agreement, or fully resolve an issue. Perhaps the mapper doesn't agree with a suggestion, and the modder isn't satisfied with their response. Alternatively, both parties might agree that although one option is better, it isn't the best option and could still be improved further with additional input. Continued feedback is the best way to improve a beatmap, so get people involved and talking to get as many different ideas, or see things from as many points of view as necessary.
+- **Treat all suggestions as valid and equally important.** Don't draw hasty conclusions about the validity of a suggestion without properly assessing it.
+- **Explain yourself as much as possible.** State what you changed, and if you didn't accept the suggestion for some reason, explain why and what you did instead.
+- **Remain calm and professional.** Say what needs to be said in order to explain yourself and leave it at that. No matter how unhelpful the suggestion was, responding with hostility, sarcasm, etc. will not help the situation.
```

Comment:
"professional" probably isn't the right word (too easy to mistake as "serious" or anti-fun imo), the message is good though. actually considering what the rest of this line explains, I think just "Remain calm." or similarly "Avoid escalating" or something like this would be fine.

Reply:
Good point. *Remain calm* by itself sounds a bit dry, but the purpose of the first bolded sentences was to be concise and descriptive, so I see the value in just getting rid of it.